webpack/fills_index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SystemStatuses from './components/extensions/about';
+import { addGlobalFill } from 'foremanReact/components/common/Fill/GlobalFill';
+
+import { registerReducer } from 'foremanReact/common/MountingService';
+import extendReducer from './components/extensions/reducers';
+
+registerReducer('katelloExtends', extendReducer);
+
+addGlobalFill('aboutFooterSlot', '[katello]AboutSystemStatuses', <SystemStatuses key="abc" />, 300);
```

Comment:
I do not mean to bike-shed this too much, but 'abc' does not strike me as a meaningful [domain related](https://github.com/theforeman/foreman/pull/6849#discussion_r296691728) value.

Reply:
oh, I forgot to change it ! thanks :)